packages/cli/README.md

```diff
@@ -1,66 +1,478 @@
 # Command Line Interface
 
-The Redwood CLI comes with RedwoodJS (which means no extra software to install!).
+<!-- toc -->
+- [Purpose and Vision](#Purpose-and-Vision)
+- [Package Lead](#Package-Lead)
+- [Roadmap](#Roadmap)
+- [Contributing](#Contributing)
+- [FAQ](#FAQ)
 
-## Usage
+## Purpose and Vision
 
-The [`yarn`](https://classic.yarnpkg.com/en/docs/install) package is required to use the Redwood CLI.
+Redwood provides a first-class CLI that helps you at every stage of development, from your first commit to your first deploy. And it comes with Redwood, which means no extra software to install!
 
-Be sure to prefix all Redwood CLI commands with `yarn`. For example, `yarn redwood new`.
+Redwood uses [yarn workspaces](https://yarnpkg.com/features/workspaces) to separate your app's sides, generators give you a smooth DX, and Prisma to manage your database. We have a generator for nearly everything, on both sides of the app, from components to functions.
```

Comment:
```suggestion
Redwood uses [yarn workspaces](https://yarnpkg.com/features/workspaces) to separate your app's sides, generators that give you a smooth DX, and Prisma to manage your database. We have a generator for nearly everything, on both sides of the app, from components to functions.
```

Reply:
Thanks I totally missed this. Thinking we'll go with "to" for parallelism.